recipes/server_setup.rb

```diff
@@ -108,22 +80,14 @@
       brick_count = 0
       peers = volume_values.attribute?('peer_names') ? volume_values['peer_names'] : volume_values['peers']
```

Comment:
Still needed if people decide to use the 'peer_names' attribute instead of the 'peers' attribute.